content/Django tips and good practices - Bruno Michetti.md

```diff
@@ -0,0 +1,164 @@
+# Django: tips and good practices
+
+As every software developer, I like to create maintainable code, not just for me but also for my partners.  It feels good to know that the programs you use and/or mantain, have good code quality. It feels good to know that we try to construct something easy to change, to reuse, to update, to review, etc. 
+
+I learned to program in the **Python** languange using the **Django** web and **Django REST** frameworks a few months ago, and I would like to share some tips and good practices for the development of REST APIs using those technologies. When I face a programing problem, first I try to solve it, and then I try to find out if there is a better way to reach the solution, and then I like to write that down or store that information, to share it with others, to publish that somewhere to help in the future to other person. I hope this blog reaches that goal: to help people learning Django, or at least to be used as a good example for those who want to code and have something to base on.
+
+## Don't reinvent the wheel
+Django is a good framework to develop web applications using the Python language. Is fast, secure, scalable and well documented. If you want to develop REST APIs, you can combine it with the Django REST framework, and you can have just in seconds, a base project generated by these tools. You can take a look at the documentation in these links:
+
+  
++ Django: https://www.djangoproject.com 
++ Django REST: https://www.django-rest-framework.org
+
+I think any teacher of computer science says to the students: "Don't reinvent the wheel". That is, if you face a problem in programation, that has a studied solution, it's better to use that, to take advantage of that solution, and solve the things faster. For a lot of common tasks in the web progamming, exists projects covering those tasks, and it's good to use them, and even better is using them, make contributions and propose improvements. So, if you want to create a REST API, the first tip of this blog is don't reinvent the wheel and use the mantained tools to attack general problems. The mentioned frameworks are great tools start the development of a REST API. 
```

Comment:
Here i think you mean programming instead of programation, also further in the line there is a typo in progamming

Reply:
+1

Reply:
this paragraph is too long. 
in english we use shorter paragraphs. you can split it in smaller units.